project_template/.gitignore

```diff
@@ -0,0 +1,2 @@
+/external/**/
+/build*

```

Comment:
should the .gitignore have the cmake/ directory as well?

Reply:
typically `cmake/` is used to contain any extra cmake modules that need to be checked in (i.e. source code), so you wouldn't want to add that to gitignore.

Reply:
actually it probably shouldn't include /build-aux